src/nanoemoji/write_config_for_glyph_svgs.py

```diff
@@ -31,8 +31,10 @@ def main(argv):
     descender = font["OS/2"].sTypoDescender
 
     with open(config_file, "w") as f:
-        f.write(textwrap.dedent(
-        f"""
+        f.write(
+            textwrap.dedent(
+                f"""
+        output_file = "COLR.ttf"
```

Comment:
you can indent this as much as you like (it's going to be de-dented), it looks nicer if you push it to the right so it aligns vertically with the three `"""`

Reply:
I must have had the wrong indent setup, looked aligned for me